apps/docs-app/docs/integrations/schematics/index.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 4
+---
+
+# Angular Schematics
+
+## Overview
+
+Analogjs provides a series of schematics that help automate some of the frequent tasks inside an Analog workspace, like generating a **Page**, a starting point for a **Blog** (work in progress), or generating **Posts** (work in progress). To use these schematics we can use this command:
+
+### Generate a Page
+
+This command will generate a page inside our `pages` folder with minimal configuration out of the box.
+
+```shell
+ng g @analogjs/platform:page --pathname=index --project=/
+```
+
+it also works with the Analog specific filenames, **Note: this names needs to be surrounded by single quotes** ex:
+
+```shell
+ng g @analogjs/platform:page --pathname='(blog)' --project=/
+```
+
+The schematic as well accepts subfolders to structure our project properly.
+
+```shell
+ng g @analogjs/platform:page --pathname='products/[products]' --project=/
```

Comment:
```suggestion
ng g @analogjs/platform:page --pathname='products/[products]'
```

Reply:
i tried this without the project and it generates the file in the wrong place the '/'